lib/jsdom/browser/domtohtml.js

```diff
@@ -8,7 +8,7 @@ exports.domToHtml = function (iterable) {
   let ret = "";
   for (const node of iterable) {
     if (node.nodeType === NODE_TYPE.DOCUMENT_NODE) {
-      ret += parse5.serialize(node, {
+      ret += parse5.serialize(idlUtils.tryWrapperForImpl(node), {
```

Comment:
I see, interesting. Just include that in the commit message please :). Maybe also a TODO comment about fixing the callers.